src/chrome/content/rules/upbad.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="upbad.com">
+        <target host="upbad.com" />
+        <target host="*.upbad.com" />

```

Comment:
Please avoid wildcards. Use `www.upbad.com`.
Also, our style guide requires to use tabs to intend lines.